plugins/network-elements/nuage-vsp/src/com/cloud/network/guru/NuageVspGuestNetworkGuru.java

```diff
@@ -114,7 +117,9 @@ public Network design(NetworkOffering offering, DeploymentPlan plan, Network use
         PhysicalNetworkVO physnet = _physicalNetworkDao.findById(plan.getPhysicalNetworkId());
         DataCenter dc = _dcDao.findById(plan.getDataCenterId());
         if (!canHandle(offering, dc.getNetworkType(), physnet)) {
-            s_logger.debug("Refusing to design this network");
+            if (s_logger.isDebugEnabled()) {
+                s_logger.debug("Refusing to design this network");
```

Comment:
This is <code>DEBUG</code> logging since core ACS asks our plugin to design some network which we do not support, therefore we just log that we are refusing it, and just returning <code>NULL</code> which core ACS will handle nicely. Added some additional info about the network to be designed.